src/renderer.ts

```diff
@@ -30,8 +31,8 @@ export function factory (environment: string): ErrorRequestHandler {
       : slug(err.output.payload.error, { replacement: '_', lower: true })
 
     const output = shouldDisplayErrorStack(environment) && data && data.stack
-      ? { status, error: { code, message, stack: data.stack } }
-      : { status, error: { code, message } }
+      ? { status, error: { code, message, stack: data.stack }, additionalData: err.data }
+      : { status, error: { code, message }, additionalData: err.data }
```

Comment:
I'd put `additionalData` inside the the `error` property and call it `data`.

The output would be something like this:

```json
{
  "status": 422,
  "error": {
    "code": "unprocessable_entity",
    "message": "required fields are missing",
    "data": {
      "fields": ["user.name"]
    }
  }
}
```